pkg/plugin/linuxutil/ethtool_stats_linux.go

```diff
@@ -12,11 +12,15 @@ import (
 	"go.uber.org/zap"
 )
 
+const limitValue = 2000
+
 type EthtoolReader struct {
-	l         *log.ZapLogger
-	opts      *EthtoolOpts
-	data      *EthtoolStats
-	ethHandle EthtoolInterface
+	l           *log.ZapLogger
+	opts        *EthtoolOpts
+	data        *EthtoolStats
+	ethHandle   EthtoolInterface
+	unsupported map[string]bool
+	limit       int
```

Comment:
nit: this should probably be a uint

Reply:
@timraymond just curious, why would this being an uint better than int? (besides the fact that the limit can't be a negative number 🙂)

Reply:
@nddq Yeah, that's the only reason. It says, "this would be nonsense if it was negative so we're disallowing that" (because it would be). Robert Griesemer at one point on the Go mailing list had an interesting opinion to default to `int` because its behavior is less surprising when doing arithmetic that would result in negative numbers. In this case though, the field should be immutable, so we don't have those concerns. But it's really a nit.